README.rst

```diff
@@ -84,20 +90,36 @@ Launch Sphinx in an ephemeral pex environment using the Sphinx entry point ``sph
 
     $ pex sphinx -e sphinx:main -- --help
 
-Build a standalone pex binary into ``pex.pex`` using the ``pex`` console_scripts entry point:
+Using Entry Points
+~~~~~~~~~~~~~~~~~~
+
+Projects specifying a ``console_scripts`` entry point (with
+`Flit <https://flit.readthedocs.io/en/latest/pyproject_toml.html#scripts-section>`_,
+`Poetry <https://python-poetry.org/docs/pyproject/#scripts>`_,
+or `setuptools <https://python-packaging.readthedocs.io/en/latest/command-line-scripts.html#the-console-scripts-entry-point>`_)
+in their configuration can build standalone executables for those entry points.
+
+To build a standalone ``pex-tools-executable.pex`` binary, including
```

Comment:
The `including` makes it sound like the `pex` distribution is being thrown in for some random purpose, but it's the vehicle of the `pex-tools` script. How about "To build a standalone ``pex-tools-executable.pex`` binary that runs the ``pex-tools`` console script found in all ``pex`` version ``2.1.35`` and newer distributions:"